Assets/Scripts/receiveSignal/recieveSignalSpinningBlade.cs

```diff
@@ -0,0 +1,69 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class recieveSignalSpinningBlade : receiveSignal 
+{
+
+    //private Collider2D collider2D2;
+    // private Renderer renderer2;
+   public  SpinningBlade spinningBlade;
+   public GameObject spinningSawblade;
+   public float rotationSpeed;
+   public bool isRight;
+
+
+
+    // Use this for initialization
+    void Start()
+    {
+
+
+        
+
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (isRight)
```

Comment:
avoid doing this, we change direction whenever it receives signal, so u dont have to check every updates